### PR TITLE
feat(wizard): add after signup wizard

### PR DESCRIPTION
**Pull request by VladislavSokov**
Screenshot from 2023-10-06 16-21-01
Screenshot from 2023-10-06 16-21-25
Screenshot from 2023-10-06 16-24-38

### CODE REVIEW

File: app/views/wizards/new_objective.html.slim [hunk 1]
```diff
@@ -0,0 +1,24 @@
+.columns.is-centered
+  .column.is-one-third
+    h2.title.is-4.has-text-centered Step 3
```
Comment (ka8725) [inline app/views/wizards/new_objective.html.slim:3]: image

**Review by ka8725 — COMMENTED** [review]
@VladislavSokov very great work, thanks! Added some small things to improve. I used the feedback from beta testers to create these instructions.

File: app/views/wizards/new_objective.html.slim (hunk 1 above)
Comment (ka8725) [reply app/views/wizards/new_objective.html.slim:3]: Note, "price" but not amount. No need to change in DB for now.

File: app/controllers/wizards_controller.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+                                        to_account: account,
+                                        **ps.slice(:amount))
+
+    redirect_to new_objective_wizard_path
+  end
+
+  def create_objective
+    account.objectives.create!(ps.slice(:name, :amount))
+
+    redirect_to my_account_path
```
Comment (ka8725) [inline app/controllers/wizards_controller.rb:27]: redirect to the created account please

File: app/views/wizards/new_account.html.slim [hunk 3]
```diff
@@ -0,0 +1,13 @@
+.columns.is-centered
+  .column.is-one-third
+    h2.title.is-4.has-text-centered Step 1
```
[the remark this answers is not in the document]
Comment (ka8725) [reply app/views/wizards/new_account.html.slim:3]: Title: Create an account for your child

Progress: 

1 out of 3
or
1/3

**Comment by VladislavSokov** [comment]
Screenshot from 2023-10-10 19-49-58
Screenshot from 2023-10-10 19-50-07
Screenshot from 2023-10-10 19-50-12

**Comment by ka8725** [comment]
@VladislavSokov it's looking good, please add a title to each page:

Title: Create an account for your child


it's "h1" on top on "Step X"

**Comment by ka8725** [comment]
Also please check text carefully. It's not 100% the same. Probably makes sense to find some text parser from images?

**Comment by VladislavSokov** [comment]
Screenshot from 2023-10-11 10-27-53
Screenshot from 2023-10-11 10-28-03
Screenshot from 2023-10-11 10-28-12

**Comment by ka8725** [comment]
@VladislavSokov the title "Create an account for your child" is a title for each wizard step. And each step 1,2,3 has their own subtitle. It was good before the changes you just made. I asked just add only title to each page and it's the same for every step.

**Comment by VladislavSokov** [comment]
Screenshot from 2023-10-11 14-04-34
Screenshot from 2023-10-11 14-04-40
Screenshot from 2023-10-11 14-04-45
Screenshot from 2023-10-11 14-04-48

**Review by ka8725 — APPROVED** [review]

**Pull request merged**